index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <meta name="description" content="Analyse the financial markets using beautiful visualisations and unique trading tools, powered by a fast and intuitive interface." />
     <title>SmartCharts | Binary.com</title>
-    <link rel="stylesheet" type="text/css" href="./dist/smartcharts.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="./dist/smartcharts.css?03cb1418" media="screen" />
```

Comment:
Was this intended? Why do we need to add the hash?

Reply:
This was changed during `gh-pages` deploy. I've reverted this now.